pages-people/people-marie.html

```diff
@@ -0,0 +1,79 @@
+<a name="marie fayard"></a>
```

Comment:
To keep it consistent with other profiles, please add a hyphen to make this `name="marie-fayard"`